scripts/gen_vimdoc.py

```diff
@@ -42,6 +42,7 @@
 import collections
 import msgpack
 import logging
+from typing import Tuple
```

Comment:
MIN_PYTHON_VERSION (defined below) is 3.6, looks like "typing" is available [since 3.5](https://docs.python.org/3/library/typing.html)